jujugui/static/gui/src/app/components/status/test-status.js

```diff
@@ -270,726 +73,33 @@ describe('Status', function() {
-            columnSize: 3
-          }, {
-            content: 'Consumes',
-            columnSize: 3
-          }, {
-            content: 'Type',
-            columnSize: 3
-          }]}
-          key="relations"
-          rowClasses={['status-view__table-row']}
-          rowColumnClasses={['status-view__table-column']}
-          rows={[{
-            columns: [{
-              columnSize: 3,
-              content: 'cluster'
-            }, {
-              columnSize: 3,
-              content: (
-                <a className="status-view__link"
-                  href={undefined}
-                  onClick={sinon.stub()}>
-                  <img className="status-view__icon"
-                    src="mysql.svg" />
-                  mysql
-                </a>)
-            }, {
-              columnSize: 3,
-              content: (
-                <a className="status-view__link"
-                  href={undefined}
-                  onClick={sinon.stub()}>
-                  <img className="status-view__icon"
-                    src="mysql.svg" />
-                  mysql
-                </a>)
-            }, {
-              columnSize: 3,
-              content: 'peer'
-            }],
-            key: 'rel1'
-          }, {
-            columns: [{
-              columnSize: 3,
-              content: 'website'
-            }, {
-              columnSize: 3,
-              content: (<span>wordpress</span>)
-            }, {
-              columnSize: 3,
-              content: (
-                <a className="status-view__link"
-                  href={undefined}
-                  onClick={sinon.stub()}>
-                  <img className="status-view__icon"
-                    src="ha.svg" />
-                  haproxy
-                </a>)
-            }, {
-              columnSize: 3,
-              content: 'regular'
-            }],
-            key: 'rel2'
-          }]}
-          sort={instance._byKey}
-          tableClasses={['status-view__table']} />
-      </div>
-    );
-    assert.compareJSX(wrapper.find('.status-view__content'), expected);
-  });
-
-  it('can link to the DNS address for a unit', () => {
-    const wrapper = renderComponent({ db: makeDB() });
-    const section = wrapper.find('BasicTable').at(3);
-    const column = section.prop('rows')[0].columns[4];
-    assert.equal(column.content.props.href, 'http://1.2.3.4:80');
+    const wrapper = renderComponent();
+    expect(wrapper).toMatchSnapshot();
   });
 
   it('can navigate to charms from the app list', () => {
-    const wrapper = renderComponent({ db: makeDB() });
-    const section = wrapper.find('BasicTable').at(2);
-    const column = section.prop('rows')[0].columns[4];
-    column.content.props.onClick({preventDefault: sinon.stub()});
+    const wrapper = renderComponent();
+    const section = wrapper.find('StatusApplicationList');
+    section.props().onCharmClick('u/who/django/xenial/42', { preventDefault: sinon.stub() });
```

Comment:
By providing real data to the default stubs this could use simulate or call the onclick, yeah?